### ISSUE TITLE
Font installation on Windows 7 fails 

### ISSUE THREAD

**Issue by YABruin** [Caster/Beamer-TUe#24]
Dear all, 

After following the installation instructions tue.map is not found by latex; kpsewhich tue.map does not return a location. Also, Texstudio doesn't compile anymore and complains that 'The zmbro8r source file could not be found'. I use TeXstudio 2.12.8 & MiKTeX 2.9 on a Windows 7 system. 
What am I missing?

Best regards, 
Yeshi

**Comment by Caster** [comment]
Thank you for using our Beamer style, and for the time you took to open this issue.

The last error message is caused by LaTeX not finding the font, I'm fairly sure. I certainly hope that the error does not occur when not using the TU/e style?

Have you carefully followed the installation instructions and made sure that all programs related to MiKTeX were closed when running `initexmf`? Can you confirm that the `Map tue.map` line was saved correctly (run `initexmf --admin --edit-config-file updmap`, the line should already be there)?

Honestly, I find these kind of issues hard to debug in general, but I'm certain we'll resolve this!

**Comment by YABruin** [comment]
Yes, I closed all MiKTeX related programs and Map tue.map is included in the config file. 
Unfortunately, the error now also occurs when not using the TU/e style. However, removing the Map tue.map line from the config file and the localtexmf directory from the MiKTeX path does resolve that issue.

**Comment by Caster** [comment]
Alright, that sounds to me like the mapping is working, but it cannot find the source files. Why, I don't know though. Could you verify that the paths are correct, as per [the instructions](https://caster.github.io/Beamer-TUe/#create-a-local-texmf-tree)? Also please make sure that the local TeXMF tree is the root directory, and not some subdirectory of it.

**Comment by Davidthf** [comment]
Hi, 
I have the same problem, in which "the zmbr8r source file could not be found", when trying to install the font. I'm using MikTex 2.9 and TexMaker 5.0.2 on Windows 10. After following the steps as per the instruction (also in administrator mode), running "kpsewhich tue.map" does not return location, and now the error occurs even if I don't use the TU/e style. Do you have any idea where to start?

**Comment by Davidthf** [comment]
Moving the local TeXMF tree to the MiKTeX installation folder (C:\Program Files\MiKTeX 2.9) and giving specific administrative rights to this folder solved this issue for me.

**Comment by Caster** [comment]
That's great! @YABruin, can you confirm if this works for you? I'll update the README if that is the case.

Thanks a lot for the effort, @Davidthf! Much appreciated - I don't have a Windows 10 machine to reproduce the issue on myself.
